backend/tests/helpers/utils.py

```diff
@@ -218,3 +219,13 @@ def before_record_response(response):
         return response
 
     return before_record_response
+
+
+def disable_low_level_logging(f):
```

Comment:
Instead of this, wouldn't this work https://tenacity.readthedocs.io/en/latest/#before-and-after-retry-and-logging?

Reply:
I see, my main concern is that we have to remember every time to add two decorators for this. We can add an auto-fixture for these modules that does this without us adding extra code.

Reply:
the issue is that I want to change the logging level for the assertion function only, we can try wrapping retry with this function, but I am not sure if that's a better solution

Reply:
The last option is adding a custom function that will change the level (I haven't tested it) https://tenacity.readthedocs.io/en/latest/#my_before_sleep, but then we'll need to pass so many parameters to `retry` so I am not convinced to this idea

Reply:
We can combine both functions into 1 decorator and use only this one